aspnetcore/tutorials/web-api-javascript.md

```diff
@@ -35,9 +35,9 @@ The `fetch` function returns a [Promise](https://developer.mozilla.org/docs/Web/
 
 The simplest `fetch` call accepts a single parameter representing the route. A second parameter, known as the `init` object, is optional. `init` is used to configure the HTTP request.
 
-1. Configure the app to [serve static files](/dotnet/api/microsoft.aspnetcore.builder.staticfileextensions.usestaticfiles#Microsoft_AspNetCore_Builder_StaticFileExtensions_UseStaticFiles_Microsoft_AspNetCore_Builder_IApplicationBuilder_) and [enable default file mapping](/dotnet/api/microsoft.aspnetcore.builder.defaultfilesextensions.usedefaultfiles#Microsoft_AspNetCore_Builder_DefaultFilesExtensions_UseDefaultFiles_Microsoft_AspNetCore_Builder_IApplicationBuilder_). The following highlighted code is needed in the `Configure` method of *Startup.cs*:
+1. Configure the app to [serve static files](/dotnet/api/microsoft.aspnetcore.builder.staticfileextensions.usestaticfiles#Microsoft_AspNetCore_Builder_StaticFileExtensions_UseStaticFiles_Microsoft_AspNetCore_Builder_IApplicationBuilder_) and [enable default file mapping](/dotnet/api/microsoft.aspnetcore.builder.defaultfilesextensions.usedefaultfiles#Microsoft_AspNetCore_Builder_DefaultFilesExtensions_UseDefaultFiles_Microsoft_AspNetCore_Builder_IApplicationBuilder_). The following highlighted code is needed in the *Program.cs*:
 
-    [!code-csharp[](first-web-api/samples/3.0/TodoApi/StartupJavaScript.cs?highlight=8-9&name=snippet_configure)]
+    ![Program file](first-web-api/_static/EnableDefaultMappingAndServeStaticFiles.png)
```

Comment:
We never use images for snippets. The update to .NET 6 is tracked in #23979
Can you delete the `aspnetcore/tutorials/first-web-api/_static/EnableDefaultMappingAndServeStaticFiles.PNGaspnetcore/tutorials/first-web-api/_static/EnableDefaultMappingAndServeStaticFiles.PNG` file you added and revert this change. Accept my suggestion and I'll approve this PR.

Reply:
Reverted the files as mentioned.
1. I wish to take up issue tracked in #23979 . Was the usage of images for snippets only mistake or I need to add the whole solution in order to link to code in files.

Reply:
@smandia  that would be fantastic if you could take #23979 . Let me prepare the doc for you first, give me an hour. And yes, you use a new project. In this case, I think we copy the completed project from https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/tutorials/first-web-api/samples/6.0
But I'll copy that over for you.